CovertInfiltration/Src/CovertInfiltration/Classes/X2Helper_Infiltration.uc

```diff
@@ -719,7 +719,10 @@ static function BuildFlatRisksDeck ()
 	
 	foreach default.FlatRiskSitReps(FlatRiskDef)
 	{
-		CardManager.AddCardToDeck('FlatRisks', string(FlatRiskDef.FlatRiskName));
+		if (!FlatRiskDef.InfiltrationOnly)
```

Comment:
This check needs to be done when the deck is iterated in `GetSitrepsForAssaultMission`

* `BuildFlatRisksDeck()` is shared with infil missions (hence completely removing these sitreps from usage)
* Card decks persist in the player profile, hence config changes will not always reflect (e.g. for people who use the previous version)